aurget

```diff
@@ -166,14 +168,22 @@ searching() { [[ "$opmode" == 'search' ]]; }
 
 upgrading() { [[ "$opmode" == 'upgrade' ]]; }
 
+clean() { [[ "$opmode" == 'clean' ]]; }
+
 resolving() { $resolve_dependencies; }
 
 eager_sudo() { $eager_sudo; }
 
-create_temp_directory() {
-  debug "storing temporary files in $colorM$temp_directory$nocolor"
-  mkdir -p "$temp_directory" || die 'unable to create temp directory'
-  rm -rf "${temp_directory:?}"/*
+create_package_directory() {
+  if [ -d "$pkgbuild_directory" ]
+  then
+  else
+  mkdir "$pkgbuild_directory" || die 'unable to create package directory' 
+  fi
```

Comment:
This is not formatted correctly. Please use the other `if` expressions in the file as examples.

Reply:
Sorry, I will take a look.